src/themes/teams/components/List/listItemVariables.ts

```diff
@@ -1,4 +1,7 @@
 export default siteVariables => ({
+  // Root
+  backgroundColor: siteVariables.brand,
+  fontColor: siteVariables.white,
```

Comment:
For consistency, please rename this to `color`

Reply:
Done. Also, renamed ```backgroundColor``` to ```background```.

Reply:
Oh, okay! Fixed.